### PR TITLE
Feature/cars integration

### PR DESCRIPTION
**Pull request by PVARGASM1**
Hi Guys,

Finished integration edit car and create, styles and bug-fixs and I added URL from JIRA:

https://gocabnow.atlassian.net/browse/SCRUM-76

### CODE REVIEW

File: src/components/AddNewCar/index.jsx [hunk 1]
```diff
@@ -23,85 +36,140 @@ const AddNewCar = () => {
...
         {
           headers: {
             Authorization: `Bearer ${localStorage.getItem('token')}`,
           },
-        }),
+        },
       );
-    } catch (err) {
-      console.log(err);
+      console.log(response);
```
Comment (GomezJuanEfe) [inline src/components/AddNewCar/index.jsx]: Por favor quitar console.log

File: src/components/FormEditCar/index.jsx [hunk 2]
```diff
@@ -39,178 +50,220 @@ const FormEditCar = () => {
 
   const handleSubmitUpdatedCar = (e) => {
     e.preventDefault();
+    console.log('updatedData', updatedData);
```
Comment (GomezJuanEfe) [inline src/components/FormEditCar/index.jsx]: Ojo, este también

File: src/components/FormEditCar/index.jsx [hunk 3]
```diff
@@ -39,178 +50,220 @@ const FormEditCar = () => {
...
     try {
       axios.patch(
-        `${URL}/api/cars/single`,
+        `${URL}/api/cars/${selectedCar.id}`,
         updatedData,
         { headers: { Authorization: `Bearer ${localStorage.getItem('token')}` } },
       );
+      setUpdateModal(true);
     } catch (error) {
       console.log('Error', error);
```
Comment (GomezJuanEfe) [inline src/components/FormEditCar/index.jsx]: Este sería un console.error

**Review by GomezJuanEfe — CHANGES_REQUESTED** [review]
Ojo con los console.log

File: src/components/FormEditCar/index.jsx [hunk 4]
```diff
@@ -39,178 +50,220 @@ const FormEditCar = () => {
...
-            <option>True</option>
-            <option>False</option>
-          </select>
+        <div className="container__add-inputs">
+          <label className="add_label-title"><b>Seats</b></label>
+          <br />
+          <div className="input-group">
+            <div className="container__span">
+              <span className="input-group-text">
+                <img src="https://res.cloudinary.com/dbmertsgv/image/upload/v1693273273/samples/ICONS_CAR_EDIT/seat_e37og3.png" alt="seat" />
```
Comment (ht1204) [inline src/components/FormEditCar/index.jsx:131]: Please, move this img url to a constant that could be imported and set here.

File: src/components/FormEditCar/index.jsx [hunk 5]
```diff
@@ -39,178 +50,220 @@ const FormEditCar = () => {
...
-              <img src="https://res.cloudinary.com/dbmertsgv/image/upload/v1693273273/samples/ICONS_CAR_EDIT/settings_wijv7z.png" alt="transmition" />
-            </span>
+
+        <div className="container__add-inputs">
+          <label className="add_label-title"><b>Luggage</b></label>
+          <br />
+          <div className="input-group">
+            <div className="container__span">
+              <span className="input-group-text">
+                <img src="https://res.cloudinary.com/dbmertsgv/image/upload/v1693273257/samples/ICONS_CAR_EDIT/luggage_vqprta.png" alt="luggage" />
```
Comment (ht1204) [inline src/components/FormEditCar/index.jsx:152]: same here.

File: src/components/FormEditCar/index.jsx [hunk 6]
```diff
@@ -39,178 +50,220 @@ const FormEditCar = () => {
...
-          <div className="container__span">
-            <span className="input-group-text">$</span>
+
+        <div className="container__add-inputs">
+          <label className="add_label-title"><b>Air Conditioner</b></label>
+          <br />
+          <div className="input-group">
+            <div className="container__span">
+              <span className="input-group-text">
+                <img src="https://res.cloudinary.com/dbmertsgv/image/upload/v1693273273/samples/ICONS_CAR_EDIT/snowflake_uqwwpz.png" alt="air" />
```
Comment (ht1204) [inline src/components/FormEditCar/index.jsx:173]: also here

File: src/components/FormEditCar/index.jsx [hunk 7]
```diff
@@ -39,178 +50,220 @@ const FormEditCar = () => {
...
-        </button>
-        <button type="submit" className="submit_add">Cancel</button>
-      </div>
+        <div className="container__add-inputs">
+          <label className="add_label-title"><b>Transmition</b></label>
+          <br />
+          <div className="input-group">
+            <div className="container__span">
+              <span className="input-group-text">
+                <img src="https://res.cloudinary.com/dbmertsgv/image/upload/v1693273273/samples/ICONS_CAR_EDIT/settings_wijv7z.png" alt="transmition" />
```
Comment (ht1204) [inline src/components/FormEditCar/index.jsx:198]: and here

**Review by ht1204 — CHANGES_REQUESTED** [review]
Please consider the comments above to approve, thx in advance.

**Review by danielhincapievargas — APPROVED** [review]

File: src/components/AddNewCar/index.jsx (hunk 1 above)
Comment (PVARGASM1) [reply src/components/AddNewCar/index.jsx]: Done

File: src/components/FormEditCar/index.jsx (hunk 2 above)
Comment (PVARGASM1) [reply src/components/FormEditCar/index.jsx]: Done

**Review by PVARGASM1 — COMMENTED** [review]
Changes made.

**Review by GomezJuanEfe — APPROVED** [review]

File: src/components/FormEditCar/index.jsx (hunk 4 above)
Comment (GomezJuanEfe) [reply src/components/FormEditCar/index.jsx:131]: Hi, Hector! We're going to put this on the backlog we'll do it with all the UI images that comes from Cloudinary. 👍

**Review by GomezJuanEfe — APPROVED** [review]

**Pull request merged**